src/lightning/pytorch/strategies/xla.py

```diff
@@ -225,18 +225,6 @@ def predict_step(self, *args: Any, **kwargs: Any) -> STEP_OUTPUT:
         with self.precision_plugin.predict_step_context():
             return self.model(*args, **kwargs)
 
-    def training_step_end(self, output: STEP_OUTPUT) -> STEP_OUTPUT:
-        self._pod_progress_bar_force_stdout()
-        return output
```

Comment:
TODO: where could I move this 🤔

Reply:
couldn't this go to on_training_batch_start in the following step? It might be a bit off, but should be fine I think